packages/fxa-settings/src/components/App/index.tsx

```diff
@@ -5,6 +5,7 @@
 import React, { useEffect, useState } from 'react';
 import { RouteComponentProps, Router } from '@reach/router';
 import { ScrollToTop } from '../Settings/ScrollToTop';
+import { sessionToken } from '../../lib/cache';
```

Comment:
I've imported the `sessionToken` from cache, which provides us access to the token when the user lands on the 'confirm' page from the same device but outside the React app (which is certain, the confirm page is not part of the usual React signup flow).

I'm not sure if this is the best way to go or if we have alternate options.

Reply:
Hm I'm not sure either. Does this simplify things alot?

Reply:
I'm not sure whether this simplifies things, because I'm not certain if there are other options to retrieve the `sessionToken` that I might have missed.